src/core/jvm/MemberInfo.js

```diff
@@ -6,16 +6,26 @@ import * as _ from 'lodash';
  */
 class MemberInfo {
   /**
+   * @param  {ClassInfo} classInfo - The class the given member is in
    * @param  {Number} accessFlags - access flags bit string
    * @param  {string} name - name of this entry in the ClassFile
    */
-  constructor(accessFlags, name) {
+  constructor(classInfo, accessFlags, name) {
```

Comment:
ClassInfo extends MemberInfo. not a fan of this. should only be available in MethodInfo and FieldInfo